### ISSUE TITLE
Use tagify SCSS file for styling of tag inputs

### ISSUE THREAD

**Issue by hangy** [openfoodfacts/openfoodfacts-server#10030]
### Description
We currently include the default layout of [tagify](https://github.com/yairEO/tagify/) by copying the original CSS file from the npm package [in gulp](https://github.com/openfoodfacts/openfoodfacts-server/blob/1acd2ff884e176aa2434d5527fd05f4eb7ecd717/gulpfile.ts#L136). However, the tagify package also provides a [source SCSS file](https://github.com/yairEO/tagify/blob/v4.22.2/src/tagify.scss). It would be interesting to use the SCSS file instead and adapt the styling to our new website design.

### Proposed Solution
1. Use tagifies' SCSS file as a modifiable base instead of the pre-built CSS file
1. Update the styling to match our new website design.

### Benefits
- Consistent styling with our website design.
- Easier maintenance and customization using SCSS.

### Implementation Steps
1. Create a new SCSS file, which included the tagify SCSS file.
1. Use that SCSS file instead of the CSS file in Gulp, so it's copied to `css/dist/tagify.css` during build
1. Update the styling as needed.

### Additional Notes
- Ensure compatibility with other components and styles in our project.

**Comment by IsaiahLevy** [comment]
Can I work on this?

**Comment by rakeshkushwaha332** [comment]
Hi! 👋 I’m interested in working on this issue as part of my GSoC preparation. I have experience with SCSS and build tools like Gulp, and I’d love to contribute to this task.

Could you please assign this issue to me? Thanks! 😊

**Comment by JagjeevanAK** [comment]
@rakeshkushwaha332 Any updates on this? If your work isn't complete yet, please create a draft PR. Otherwise, we can reassign the issue to someone else.

**Comment by ng169** [comment]
Hey since it has been inactive for 2 weeks, can I work on this?

**Comment by rajanarahul93** [comment]
Hi!  I’d like to take this up. I have experience with SCSS and Gulp, and I’m confident I can implement the changes as described. Assign the issue to me. Thanks!

**Comment by JagjeevanAK** [comment]
@rajanarahul93  Yup, you can go forward for this issue, If @rajanarahul93 fail's to open an PR then anyone can take this issue.

**Comment by Chetan27Pareek** [comment]
Hi! I'm interested in this issue. I have experience with CSS and I'm learning SCSS. Can I work on it?

**Comment by JagjeevanAK** [comment]
Carry, on I will assign this issue to you. If you fail to raise the PR then
anyone can take this issue.

**Comment by Aarya-Chaudhari** [comment]
Hi, I would like to work on this issue.
If there are no concerns, I will start working on a PR.
Thank you!

**Comment by Aarya-Chaudhari** [comment]
Thank you for assigning the issue to me.
I will work on this and update here.

**Comment by shaktipy** [comment]
Hi 👋,
I’d like to work on this issue: using Tagify’s SCSS file instead of the pre-built CSS and updating the styling to match the new website design.
Could you please assign it to me?  
Thanks!

**Comment by shaktipy** [comment]
Implementation for migrating Tagify from CSS to SCSS is complete and ready for review.

**Draft PR created:** [#12908](https://github.com/openfoodfacts/openfoodfacts-server/pull/12908)

### What's included:
- ✅ **SCSS migration** - Tagify now uses SCSS source instead of pre-built CSS
- ✅ **Design matching** - Updated styling with OFF brand colors (#84bd3a)
- ✅ **Responsive design** - 4 breakpoints (1024px, 768px, 480px) + touch + print
- ✅ **Build process update** - Removed tagify.css from copyCss() task
- ✅ **Windows compatibility** - Fixed SVG parsing issue in icons task

### Changes made:
1. Created `scss/_tagify.scss` with OFF design system
2. Added `@import "tagify";` to both `app-ltr.scss` and `app-rtl.scss`
3. Commented out `tagify.css` in `gulpfile.ts` copyCss() function
4. Simplified icons task for Windows compatibility
5. Added comprehensive responsive design

All requirements from the issue description have been addressed. The PR is marked as draft and ready for review when convenient.
